src/ApiKitClient.ts

```diff
@@ -28,9 +36,13 @@ export class ApiKitClient {
     });
 
     this.instance.interceptors.response.use((response: AxiosResponse) => response, (error: AxiosError) => {
-      if (error.response?.status === 401 && this.unauthorizationCallback) {
+      const { status } = error.response;
+      if (status === 401 && this.unauthorizationCallback) {
         this.unauthorizationCallback();
       }
+      if (this.globalErrorCallback) {
```

Comment:
should it be `elseif` ?

Reply:
Yes you are right, I started with like this at 1st but I changed it accidentally later!

Reply:
Done! ✅